testing/dbginfo/dbginfo.c

```diff
@@ -0,0 +1,341 @@
+/****************************************************************************
+ * apps/testing/dbginfo/dbginfo.c
+ *
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.  The
+ * ASF licenses this file to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance with the
+ * License.  You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.  See the
+ * License for the specific language governing permissions and limitations
+ * under the License.
+ *
+ ****************************************************************************/
+
+/****************************************************************************
+ * Included Files
+ ****************************************************************************/
+
+#include <stdbool.h>
+#include <stdio.h>
+#include <string.h>
+#include <sys/param.h>
+#include <unistd.h>
+
+#include <nuttx/config.h>
+#include <nuttx/version.h>
+
+#include "sysinfo.h"
+
+/****************************************************************************
+ * Private Data
+ ****************************************************************************/
+
+/* Valid Command Line Arguments */
+
+static const char *g_valid_args[] =
+{
+  "-h",
+  "--help",
+  "-n",
+  "--nuttx-os",
```

Comment:
should we remove nuttx- prefix and use host- prefix to

Reply:
Do you mean something like:
```
  "-h",
  "--help",
  "-n",
  "--os",
  "-f",
  "--flags",
  "-c",
  "--config",
  "-o",
  "--host-os",
  "-p",
  "--host-path",
  "-k",
  "--host-packages",
  "-m",
  "--host-modules",
  "-v",
  "--vendor-specific",
  "--all"
```

Reply:
Yes